doc/deployment-integration/local-deployment/checkRequirements.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
```

Comment:
Do we really need this script? It needs to be maintained(for example keeping the rancher config up to date with the k3s plugin). Shouldn´ t  a somewhat sane developer(essentially every one that made it out of the first year of university) be able to check the tools themself?
In the end, somebody might ask  what bash is;)